Userland/Libraries/LibGL/GLContext.h

```diff
@@ -81,7 +81,6 @@ struct ContextParameter {
 struct VertexAttribPointer {
     GLint size { 4 };
     GLenum type { GL_FLOAT };
-    bool normalize { true };
```

Comment:
Should we bother to remove this? It's as per the spec, and will return with `glVertexAttribPointer()`

https://registry.khronos.org/OpenGL-Refpages/gl4/html/glVertexAttribPointer.xhtml

```cpp
void glVertexAttribPointer(	GLuint index,
 	GLint size,
 	GLenum type,
 	GLboolean normalized,
 	GLsizei stride,
 	const void * pointer);
```

This struct is just the "exploded" version of the arguments list from this function.

Reply:
Ah, so we don't have `glVertexAttribPointer` implemented which is why I missed that. I'll implement that API then instead of removing the normalize attribute :-)

Reply:
I had to do a double take on that one too; I could've sworn we did.

Reply:
We have `glVertexPointer`, but not its shiny new cousing `glVertexAttribPointer` :^)

Reply:
@Quaker762 check this out:
https://stackoverflow.com/questions/27017824/whats-the-purpose-of-glvertexpointer

What I understand is that `glVertexPointer` is simply the old, fixed function version of pointing to vertex location data, while `glVertexAttribPointer` is the generalized way of setting data at arbitrary vertex attribute indices. I don't think they modify the same client pointer, and I think `normalize` should still be removed from the `glVertexPointer` variant. Thoughts?

Reply:
Yeah with `glVertexAttribPointer` you can specify whether you want to normalize integral attributes or not. Didn't know this was implicit with the old style APIs!